src/Illuminate/Console/ConfirmableTrait.php

```diff
@@ -15,6 +15,10 @@
      */
     public function confirmToProceed($warning = 'Application In Production!', $callback = null)
     {
+        if (! $this->input->isInteractive()) {
+            return true;
```

Comment:
`--non-interactive --force` together should have that behaviour, but not just `--non-interactive`